cli_tools/gce_windows_upgrade/upgrade_script_2008r2_to_2012r2.ps1

```diff
@@ -0,0 +1,54 @@
+#  Copyright 2020 Google Inc. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+$script:install_media_drive = ''
+
+try {
+  Write-Host 'GCEMetadataScripts: Beginning upgrade startup script.'
```

Comment:
When running from metadata scripts this log line would be redundant

Reply:
In the workflow, I wait on this msg to ensure the machine is not dead after reboot.

Reply:
Can you explain this a bit more? Are you sometimes depending on the agent to emit it, but in other cases your script to emit it?

Can you just make it explicit, and not try to emulate the agent?